bin/auth0_scaffold.js

```diff
@@ -0,0 +1,26 @@
+var Chalk = require('chalk');
+var Cli = require('structured-cli');
+var _ = require('lodash');
+var keyValList2Object = require('../lib/keyValList2Object');
+var auth0Extensions = require('./auth0_extensions');
+var extensionTypes = Object.keys(auth0Extensions).sort();
+
+
+module.exports = Cli.createCommand('scaffold', {
+    description: 'Scaffold the Auth0 hook code',
+    optionGroups: {
```

Comment:
Later we could consider an `-o <filename>` parameter so that console piping understanding is not a prerequisite :)